webapp/sources/rudder/rudder-web/src/main/elm/sources/Groups/View.elm

```diff
@@ -33,7 +33,13 @@ view model =
         [ i[class "jstree-icon jstree-ocl"][]
         , a[class ("jstree-anchor"++classDisabled {- ++classFocus -}), href (getGroupLink model.contextPath item.id.value), onClick (OpenGroupDetails item.id)]
           [ i [class "jstree-icon jstree-themeicon fa fa-sitemap jstree-themeicon-custom"][]
-          , span [class "treeGroupName", title (buildTooltipContent item.name item.description)]
+          , span
+            [
+              class "treeGroupName"
+              , attribute "data-bs-toggle" "tooltip"
+              , attribute "data-bs-placement" "right"
+              , title (buildTooltipContent item.name item.description)
+            ]
```

Comment:
We need to check that the Group actually has a description, so that we can add these attributes only if it does (and avoid initialising empty tooltips).